lib/ui/app/entities/multiple_entity_actions_dialog.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
```

Comment:
Can you explain why this class is needed/would it be possible to re-use the existing entity_actions_dialog.dart?